SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -47,6 +47,23 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 			const FRepLayoutCmd& Cmd = Changes.RepLayout.Cmds[HandleIterator.CmdIndex];
 			const FRepParentCmd& Parent = Changes.RepLayout.Parents[Cmd.ParentIndex];
 
+#if TRACE_LIB_ACTIVE
+			if (USpatialLatencyTracer* LatencyTracer = USpatialLatencyTracer::GetTracer(NetDriver))
+			{
+				TraceKey TraceId = LatencyTracer->GetTraceKey(Object, Cmd.Property); 
+				if (TraceId == USpatialLatencyTracer::InvalidTraceKey)
+				{
+					// Possibly sending a nested property? 
+					TraceId = LatencyTracer->GetTraceKey(Object, Parent.Property);
```

Comment:
What happens if you attempt to track something like an array, which has multiple leaf properties?

Reply:
I think the properties are flattened so there is only at most 2 layers of nesting - do you know if this is the case?

Reply:
I'm more interested in what happens if I try to track an array, and then update multiple elements in that array. What's the expected behaviour?

Reply:
What happens here is that GetTraceKey removes the internal mapping so this code will only trigger once. This seems fine because it's only the UProperty that is tracked we are interested in